de-lib/src/main/java/org/iplantc/de/apps/client/AppsListView.java

```diff
@@ -105,8 +109,11 @@
         void onAppNameSelected(Splittable appSplittable);
 
         @SuppressWarnings("unusable-by-js")
-        void onAppInfoSelected(Splittable appSplittable,
-                               boolean showQuickLaunchFirst);
+        void onAppInfoSelected(String appId,
+                               String systemId,
+                               boolean isPublic,
+                               boolean showQuickLaunchFirst,
+                               Splittable app);
```

Comment:
The required parameters for details to work properly are `appId`, `systemId` and `isPublic`. But we need to pass in the `app` splittable for now since the details view doesn't have access to selected `app` in react. Once all parts of the Apps window is in react, we can remove this parameter. Details view will get the selected `app` from the the state.